spec/post_nl/operation_spec.rb

```diff
@@ -1,37 +0,0 @@
-# frozen_string_literal: true
-
-require "spec_helper"
-
-RSpec.describe PostNL::Operation do
```

Comment:
This was basically just an example from the template, that we can copy into the individual operations classes (which I've done here)

There's no real value in keeping it now as we would never call this class directly.